### PR TITLE
Add an initial Japanese translation

### PR DESCRIPTION
**Pull request by hnakamur**
Hi, I made an initial Japanese translation.
This is untested, rough-cut version.
However this may be a good starting point.

Could you review this pull request?
Thanks!

### CODE REVIEW

**Comment by stgraber** [comment]
Hi,

Thanks for that work!

Note that all commits in the pull request must be Signed-off with the usual Signed-off-by line, Travis is reporting that one or more commits in your branch aren't.

Can you fix that and maybe squash some of your commits together to make the branch look a bit cleaner (ideally getting rid of those merge commits)?

Thanks!

**Comment by hnakamur** [comment]
@stgraber Thanks for your comment. I rebased and cleaned up `initial_japanese_translation` branch.

File: po/ja.po [hunk 1]
```diff
@@ -0,0 +1,662 @@
...
+"Project-Id-Version: LXD\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-03-12 13:58+1100\n"
+"PO-Revision-Date: 2015-02-26 02:05-0600\n"
+"Last-Translator: Stéphane Graber <stgraber@ubuntu.com\n"
+"Language-Team: French <fr@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
```
Comment (stgraber) [inline po/ja.po]: The header should be updated for your translation. At least Last-Translator, Language-Team and possibly PO-Revision-Date (not very important for that one).

File: po/ja.po [hunk 2]
```diff
@@ -0,0 +1,662 @@
+# French translation for LXD
```
Comment (stgraber) [inline po/ja.po]: I'm pretty sure this isn't French :)

**Comment by stgraber** [comment]
Thanks!

Just two small problems to fix with the .po and I'll merge.

**Comment by hnakamur** [comment]
Thanks for pointing out problems. I just fixed them.
Could you take another look?
Thanks!

**Comment by stgraber** [comment]
Perfect, thanks!

**Comment by hnakamur** [comment]
Thanks for merging!

**Pull request merged**